pipeline/lovd/separateFunctionalAnalysisTechniqueAndResult.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+"""
+Zack Fischmann
+9/21/2018
+
+Description:
+    LOVD functional analysis technique and result are reported in a colon delimited fashion
+    e.g. technique:result. This file splits them into two separate fields.
+"""
+
+import argparse
+import sys
+import os
+import urllib
+from collections import defaultdict
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help='Input LOVD file for ammendment.')
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        help='Ouput txt file result.')
+    options = parser.parse_args()
+    return options
+
+
+def main(args):
+    options = parse_args()
+    f_in = options.input
+    f_out = options.output
+
+    headerline = f_in.readline()
+
+    # add functional analysis fields
+    ammendedHeaderline = headerline.split('\r\n')[0] + '\t' + '"functional_analysis_technique"' + '\t' + '"functional_analysis_result"' + '\r\n'
```

Comment:
It fields like leveraging a library like Pandas could shorten the code quite a bit and make it clearer. In particular, one wouldn't have to deal too much with the specifics of parsing/generating csv/tsv files.

Reply:
I've never used pandas before but it looks like an obvious improvement. I'll update it if time permits :)